src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/drools/AllServicesOrgRoleTest.java

```diff
@@ -1069,4 +1069,91 @@ void shouldRejectIacOrgRequestedRoleForHearingFromAnotherJurisdiction(String rol
         });
     }
 
+    @ParameterizedTest
```

Comment:
Please move to new test class to reflect that it is configured in new PRM drool file (plus this one is getting a little full)

Reply:
moved to own class `PrmOrgRoleTest` commit: [f8269a9](https://github.com/hmcts/am-role-assignment-service/pull/2062/commits/f8269a94896e32c2d336b6c76aa7ff585d45c1b5)